scanner/src/lib/dataproducts/Battery_ManufacturingDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,251 @@
+<script lang="ts">
+  /*
+    Docs:
+
+    https://github.com/ioxio-dataspace/sandbox-definitions/blob/main/src/DigitalProductPassport/Battery/ManufacturingDataSheet_v0.1.py
+
+    https://definitions.sandbox.ioxio-dataspace.com/definitions/DigitalProductPassport/Battery/ManufacturingDataSheet_v0.1
+
+    https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_Battery_ManufacturingDataSheet_v0_1_DigitalProductPassport_Battery_ManufacturingDataSheet_v0_1_post
+     */
+
+  import { countryListAlpha3, formatNumber } from "$lib/common"
+  import DataRow from "$components/DataRow/index.svelte"
+  import True from "$assets/true-circle.svg"
+  import False from "$assets/false-circle.svg"
```

Comment:
Please use clearer import names, like `TrueIcon`, `FalseIcon`, check other files for any patterns - I think we might use `Svg`?